amqp/channel.py

```diff
@@ -1786,7 +1787,10 @@ def confirm_handler(method, *args):
             self.confirm_select()
         ret = self._basic_publish(*args, **kwargs)
         # Waiting for confirmation of message.
-        self.wait([spec.Basic.Ack, spec.Basic.Nack], callback=confirm_handler)
+        timeout = confirm_timeout or kwargs.get('timeout', None)
```

Comment:
`confirm_timeout` is not enough ? A more global timeout can be sent here ?

Reply:
You can either use the timeout for publishing or pass a specific timeout for waiting.
I'd rather avoid not having timeouts as much as possible.

Reply:
Okay, it LGTM ! I let @matusvalo approving